scripts/x-data-grid-premium.exports.json

```diff
@@ -306,6 +306,7 @@
   { "name": "gridExpandedRowCountSelector", "kind": "Variable" },
   { "name": "gridExpandedSortedRowEntriesSelector", "kind": "Variable" },
   { "name": "gridExpandedSortedRowIdsSelector", "kind": "Variable" },
+  { "name": "gridExpandedSortedRowTreeLevelPositionLookupSelector", "kind": "Variable" },
```

Comment:
Perhaps we can keep these new selectors unexported for now to avoid increasing the API surface?

Reply:
Is this used for the docs reference?
I see that all selectors are listed. This one has `@ignore` already (as some others in the list)

Reply:
While not documented, it's still exported from the package, technically being a public API.
If there's no need to export it, I suggest not doing so.